Validation/MuonME0Validation/plugins/MuonME0DigisHarvestor.cc

```diff
@@ -119,7 +119,7 @@ TH1F* MuonME0DigisHarvestor::ComputeBKG(TH1F* hist1, TH1F* hist2, std::string na
     }
 
     int nEvts = hist2->GetEntries();
-    float scale = 6*2*nEvts*9*25e-9;
+    float scale = 6*2*nEvts*3*25e-9;
```

Comment:
at the moment this information can be taken from here https://github.com/cms-sw/cmssw/blob/master/SimMuon/GEMDigitizer/python/muonME0ReDigis_cfi.py#L11 , when we move to the realistic digitizer from similar lines